site/packages/aa-signers/arcana-auth/introduction.md

```diff
@@ -0,0 +1,92 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: ArcanaAuthSigner
+  - - meta
+    - name: description
+      content: Overview of the ArcanaAuthSigner class in aa-signers
+  - - meta
+    - property: og:description
+      content: Overview of the ArcanaAuthSigner class in aa-signers
+---
+
+# Arcana Auth Signer
+
+`ArcanaAuthSigner` is a signer implementation that extends `SmartAccountAuthenticator` to leverage the [Arcana Auth SDK](https://docs.arcana.network). It supports features such as authentication, message and typed data signing, and authentication details retrieval.
+
+`ArcanaAuthSigner` provides implementations for all methods on `SmartAccountAuthenticator`:
+
+1.  [`authenticate`](/packages/aa-signers/arcana-auth/authenticate) -- supports user authentication.
+2.  [`getAddress`](/packages/aa-signers/arcana-auth/getAddress) -- supports typed data signatures from the smart contract account's owner address.
+3.  [`signMessage`](/packages/aa-signers/arcana-auth/signMessage) -- supports message signatures.
+4.  [`signTypedData`](/packages/aa-signers/arcana-auth/signTypedData) -- supports typed data signatures.
+5.  [`getAuthDetails`](/packages/aa-signers/arcana-auth/getAuthDetails) -- supports authentication details retrieval.
+
+## Install Dependencies
+
+`ArcanaAuthSigner` requires the installation of the [`Arcana Auth`](https://www.npmjs.com/package/@arcana/auth) SDK. `aa-signers` lists it as an optional dependency.
+
+::: code-group
+
+```bash [npm]
+npm i -s @arcana/auth
+```
+
+```bash [yarn]
+yarn add @arcana/auth
+```
+
+## Usage
+
+::: code-group
+
+```ts [example.ts]
+import { ArcanaAuthSigner } from "@alchemy/aa-signers";
+
+const newArcanaAuthSigner = new ArcanaAuthSigner({
+  clientId: "ARCANA_AUTH_CLIENT_ID",
+});
+// or use inner
+
+// import { AuthProvider } from "@arcana/auth";
+
+// const authParams = {
+//  theme: 'light',
+//  network: 'testnet',
+//  position: 'left'
+// }
+
+// const inner = new AuthProvider("ARCANA_AUTH_CLIENT_ID", {authParams};
+// const newArcanaAuthSigner = new ArcanaAuthSigner({inner});
```

Comment:
feel free to un-comment this for readability!

Reply:
Done.